### DISCUSSION TITLE
QUICK performance on RTXA5000

### DISCUSSION THREAD

**Discussion by medchemunipr** [merzlab/QUICK#289] [Q&A]
Hello everybody,
my group and I are planning to extend our academic cluster facility with a GPU node to extensively use Quick, and we would like to buy RTXA5000, so I would ask if anybody ever tested the performance of Quick with this type of card.
Thank you very much for your help!
Laura

**Comment by Madu86** [comment]
Hi @laurascalvini, Thanks for reaching out. Its great to hear that you are interested to use QUICK. Unfortunately, we never had access to RTX A5000. I do have a RTX3080 Ti and based on tech specs (https://www.techpowerup.com/gpu-specs/rtx-a5000.c3748, https://www.techpowerup.com/gpu-specs/geforce-rtx-3080-ti.c3735) I can only estimate that performance on A5000 would be at least %20 worse than this card. Here are some timings for Valinomicin B3LYP/6-31G** gradient calculation example (168 atoms, 1620 basis functions) on different NVIDIA cards. 

A100 (Data center card, Ampere micro architecture): 167 s
V100 (Data center card, Volta micro architecture) : 266 s
RTX3080 Ti (Gaming card, Ampere micro architecture): 351 s

The QUICK version that Ive used here is the latest master from Github repo. Let us know if you have any questions.
